inotify/adapters.py

```diff
@@ -323,6 +324,14 @@ def event_gen(self, ignore_missing_new_folders=False, **kwargs):
 
             yield event
 
+    def _load_trees(self, path_list: List[pathlib.Path]):
+        for entry_path in path_list:
+            if entry_path.is_dir():
+                self._i.add_watch(str(entry_path), self._mask)
+                for sub_entry_path in entry_path.rglob('*'):
```

Comment:
Let's just stick with `listdir()`, which is going to be much more efficient.

Also, it'd be better to do (for example):
```
if entry_path.is_dir() is False:
    continue
```

..which will reduce nesting.